### PR TITLE
Newmana/read compare vector tiles

### PR DESCRIPTION
**Pull request by newmana**
I couldn't find a withTempFile.

### CODE REVIEW

**Comment by newmana** [comment]
Closes #37 and #25

**Comment by newmana** [comment]
Tests pass on Linux.

image

File: test/TileSpec.hs [hunk 1]
```diff
@@ -1,18 +1,83 @@
...
+generateMvtFile geoJsonFile layerName coords = do
+  bs <- LBS.readFile geoJsonFile
+  let ebs = eitherDecode bs :: Either String GeoJson
+      decodeError = error . (("Unable to decode " <> geoJsonFile <> ": ") <>)
+      geoJson = either decodeError id ebs
+  pluginDir <- fromMaybe "/usr/local/lib/mapnik/input" <$> lookupEnv "MAPNIK_PLUGINS_DIR"
+  et <- MapboxVectorTile.fromGeoJSON defaultTileSize geoJson layerName pluginDir coords
+  either (error . unpack . ("Failed to create tile: " <>)) (pure . fromStrict) et
+
+withTempFile :: (FilePath -> IO a) -> IO a
```
Comment (ajmcmiddlin) [inline test/TileSpec.hs]: https://hackage.haskell.org/package/temporary

**Review by ajmcmiddlin — COMMENTED** [review]
Looks good - don't know if you want to use `temporary` package rather than your own `withTempFile`

**Pull request merged**